src/components/cards/controltable/ControlRowCol.vue

```diff
@@ -35,7 +35,7 @@
         </template>
       </v-clamp>
     </v-col>
-    <v-col v-else lg="6" xl="6" sm="6" md="6" cols="12" class="left">
+    <v-col v-else lg="5" xl="5" sm="6" md="6" cols="12" class="left">
```

Comment:
This can be replaced with `<v-col v-else cols="12" sm="6" lg="5"  class="left">`

Although I'm not sure I understand the `lg="5"` since that will leave a blank 2 columns on large and xl screens I believe.-